snuba/optimize.py

```diff
@@ -22,12 +22,21 @@ class JobTimeoutException(SerializableException):
     pass
 
 
+def _get_metrics_tags(table: str, clickhouse_host: Optional[str]) -> Mapping[str, str]:
+    return (
+        {"table": table, "host": clickhouse_host}
+        if clickhouse_host
```

Comment:
I'm not saying it's wrong but why would `clickhouse_host` ever be nil?

Reply:
Good question. I found out that the replacement consumer can also call run_optimize under certain circumstances. https://github.com/getsentry/snuba/blob/master/snuba/replacer.py#L418-L420. From there, `clickhouse_host` won't be passed.